core/src/tile/tileWorker.cpp

```diff
@@ -144,6 +185,10 @@ void TileWorker::setScene(std::shared_ptr<Scene>& _scene) {
     }
 }
 
+void TileWorker::notifyAll() {
+    m_condition.notify_all();
+}
+
```

Comment:
I thought that the worker will wait on the conditional variable until
1. condition is modified and satisfied
2. "and" a notification is sent.
http://en.cppreference.com/w/cpp/thread/condition_variable

Because in this case, `m_queue` will not be empty and if it is not required of the conditional variable to be notified to check for wait condition, then I agree we do not need to send the notification. But according to the description of conditional variable, it seems a notification has to be sent explicitly. Correct me here if I am misinterpreting conditional variable usage.